examples/step-71/doc/results.dox

```diff
@@ -0,0 +1,167 @@
+<h1>Results</h1>
+
+We run the program with a right hand side that will produce the
+solution $u = \sin(\pi x) \sin(\pi y)$ and with clamped
+boundary conditions in the domain $\Omega = (0,1)^2$.
+We test this setup using $Q_2$, $Q_3$, and $Q_4$ elements, which one can
+change via the `fe_degree` variable in the `main()` function. With mesh
+refinement, the $L_2$ convergence rates, $H_1$-seminorm convergence
+and $H_2$-seminorm convergence of $u$
+should then be around 2, 2, 1 for $Q_2$; 4, 3, 2 for
+$Q_3$; and 5, 4, 3 for $Q_4$, respectively.
+
+From the papers by Brenner et al., it is not immediately clear what
```

Comment:
This is what I found:
- Brenner/Gudi/Sung: ">=1", "we take the penalty parameter \sigma to be 5"
- Fenics/dolfin tutorial: "alpha=8"

Reply:
and Wells/Dung 2007 for Kirchhoff plates: stability with " > n_e (number of edges belonging to an element)", section 4.2